ChatQnA/docker/gaudi/README.md

```diff
@@ -2,6 +2,83 @@
 
 This document outlines the deployment process for a ChatQnA application utilizing the [GenAIComps](https://github.com/opea-project/GenAIComps.git) microservice pipeline on Intel Gaudi server. The steps include Docker image creation, container deployment via Docker Compose, and service execution to integrate microservices such as embedding, retriever, rerank, and llm. We will publish the Docker images to Docker Hub, it will simplify the deployment process for this service.
 
+## Quickstart
+
+Clone the repo
+
+```bash
+> git clone https://github.com/opea-project/GenAIExamples.git
+```
+
+Navigate to this directory
+
+```bash
+> cd GenAIExamples/ChatQnA/docker/gaudi
+```
+
+Generate a default `.env` file.
+
+```bash
+> bash gen_env.sh
```

Comment:
If you deleted the `set_env.sh` file, then all of the `get_env.sh` and `README` files should be updated at the same time for consistency.

Reply:
Do you mean it needs to be changed in every example folder?